aws/resource_aws_lambda_event_source_mapping_test.go

```diff
@@ -48,11 +48,13 @@ func TestAccAWSLambdaEventSourceMapping_Kinesis_basic(t *testing.T) {
 				PlanOnly: true,
 				Config:   testAccAWSLambdaEventSourceMappingConfigKinesisBatchSize(rName, "null"),
 			},
-			{
-				ResourceName:      resourceName,
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
+			/*
```

Comment:
This test will fail if the alphabetical order of the subnets is different than how AWS sorts them.